exercises/concept/leap/.approaches/introduction.md

```diff
@@ -0,0 +1,41 @@
+# Introduction
```

Comment:
I've noticed that this is now a concept exercise. Should it actually be so as Julia doesn't have a Syllabus yet?